temboardagent/plugins/monitoring/db.py

```diff
@@ -0,0 +1,84 @@
+# coding: utf-8
+
+import json
+import os
+import sqlite3
+from textwrap import dedent
+
+
+def bootstrap(path, dbname):
+    """Create SQLite database model we use to store stuff."""
+
+    with sqlite3.connect(os.path.join(path, dbname)) as conn:
+        c = conn.cursor()
+        c.execute("DROP TABLE IF EXISTS last_measures")
```

Comment:
Can you explain the logic for this `DROP TABLE`? Does this table somehow behaves like a volatile queue (emptied upon agent restart)?

In fact, it might be worth explaining the SQL model somewhere (inline here or in the docstring of the module or of the bootstrap() function) for future reference.

Reply:
I've put some explanations in bootstrap() docstring